### ISSUE TITLE
metadata map is hardcoded to the Virginia state boundary

### ISSUE THREAD

**Issue by ChristopherTracey** [PNHP/Regional_SDM#23]
Here in the .rnw

```
plot(spTransform( referenceBoundaries[referenceBoundaries$STATE_NAME != "Pennsylvania",]
                  , CRSobj = results_shape@proj4string), add=T, border="gray50", lwd=0.3, col = "white")
plot(spTransform( referenceBoundaries[referenceBoundaries$STATE_NAME == "Pennsylvania",]
                  , CRSobj = results_shape@proj4string), add=T, border="black", lwd=0.7)
```

We should make this more flexible.

**Comment by ChristopherTracey** [comment]
Also, add in a variable for this in the figure caption for the state name as well.

**Comment by dnbucklin** [comment]
fixed ed26f7f34a6e254a9a4869c1ca34750a98bc90e3. 

The code still assumes that:
`nm_refBoundaries` : a shapefile of all states with the column `STATE_NAME`
`nm_studyAreaExtent` : a shapefile of state(s) in the study area with the column `STATE_NAME`. A subset of `nm_refBoundaries`. These states will be outlined in black in Figure 4 and listed in the caption.
